src/views/MainPage/MainPage.js

```diff
@@ -1,19 +1,28 @@
-import Button from '../../components/Button/Button.js';
 import './MainPage.css';
+import { retrieveAnimalFact } from '../../api/FactsController.js';
+import { Fact } from '../../model/Fact.js';
+import Button from '../../components/Button/Button.js';
 import { onNavigationChange } from '../../components/router/Router.js';
 
-const MAIN_ANIMAL_PATH = './kangoroo.png';
-const FACT_ANIMAL_URL = 'http://placekitten.com/400/500';
-const ADOPTION_ANIMAL_URL = 'http://placekitten.com/300/400';
+const MAIN_ANIMAL = { path: './kangoroo.png', alt: 'kangoroo that tells informations' };
+const FACT_ANIMAL = {
+  url: 'http://placekitten.com/400/500',
+  alt: 'the animal the sentence is about',
+  fallbackFact:
+    "Cat's eyes shine in the dark because of the tapetum, a reflective layer in the eye, which acts like a mirror.",
+};
+const ADOPTION_ANIMAL = { url: 'http://placekitten.com/300/400', alt: 'the animal to adoption' };
 
 export function renderMainPage() {
+  const newFact = new Fact();
   renderMainView();
-  renderAnimalFact();
+  renderNavigation();
+  renderAnimalFact(newFact);
 }
 
 function renderMainView() {
   document.querySelector('#app').innerHTML = `
-  <div class="container">
+  <div class="container main-page-container">
```

Comment:
If you change the selector name, remember to remove unused class names. It will be easier to avoid future conflicts in the CSS

Reply:
Thank you for noticing, but at this point this class was not unused because I also used it on a page in the second issue I'm working on.